src/backend/appwrite.config.ts

```diff
@@ -2,7 +2,7 @@
 import { Account, Client, Databases, Storage, ID, Query } from "appwrite";
 
 const appwriteClient = new Client();
-
+//https://cloud.appwrite.io/v1
```

Comment:
please remove this comment. Please don't put unnecessary comments.

Reply:
hey i resolved all the issues(cheked whether both the passwords are matching or not ,removed package.json file ,removed unnecessary comments,password is atleast of length 8 , contains atleast one digit ,atleast one alphabet ,
atleast one special character.I used regex for checking password strength.
I checked these things on the frontend.I didnot check it backend wise.
If u still find any issues let me know.